pkgs/by-name/rc/rcp/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
```

Comment:
`stdenv`, `buildPackages`, `pkg-config` are apparently not required, I guess they can be removed.